src/components/editor/feature-action-panel.js

```diff
@@ -76,7 +76,7 @@ export class FeatureActionPanel extends PureComponent {
         }}
       >
         <ActionPanel>
-          <ActionPanelItem label="layers" Icon={Layers}>
+          <ActionPanelItem className="editor-layers-list" label="layers" Icon={Layers}>
```

Comment:
when passing classnames to styled componenet you can use the `attrs` method

```js
const StyledItem = styled.div.attrs(props => ({
  className: classnames('action-items', props.className)
}))`
  display: flex;
  flex-direction: row;
`
```

Reply:
this is a good approach when I want to have a base class name for a given styled component.

I think in this case, since i don't need a base CSS class, styled-components already take care of things automatically by simply passing className property.